### PR TITLE
PTH-719: Added product & team name in the report metadata

### PR DESCRIPTION
**Pull request by mohammad-sahil-ev**
Added product & team name in the report metadata

### CODE REVIEW

**Review by mydailypuff — APPROVED** [review]

File: src/advisor/reports/html_report.py [hunk 1]
```diff
@@ -44,6 +44,8 @@ def write_items(self, output_file, items):
         branch_name = self.branch_name
         report_status = self.report_status
         savings = self.savings
+        productname = self.productname
```
Comment (adubey-8720) [inline src/advisor/reports/html_report.py:47]: non-pythonic variable name. should be product_name and team_name

File: src/advisor/reports/report.py [hunk 2]
```diff
@@ -36,6 +36,8 @@ def __init__(self, root_directory, branch, savings, target_os='linux'):
         self.branch_name = branch
         self.report_status = ''
         self.savings = savings
+        self.productname = productname
```
Comment (adubey-8720) [inline src/advisor/reports/report.py:39]: Same as above

File: src/advisor/reports/report_factory.py [hunk 3]
```diff
@@ -53,12 +53,12 @@ def output_format_for_extension(self, extension):
         """ Choose an output format based on the given file name extension. """
         return ReportFactory._OUTPUT_FORMAT_FOR_EXTENSION.get(extension.lower(), None)
 
-    def createReport(self, root_directory, branch, savings, target_os='linux', issue_type_config=None, output_format=ReportOutputFormat.TEXT):
+    def createReport(self, root_directory, branch, savings, productname, teamname, target_os='linux', issue_type_config=None, output_format=ReportOutputFormat.TEXT):
```
Comment (adubey-8720) [inline src/advisor/reports/report_factory.py:56]: non pythonic function name

create_report

**Review by adubey-8720 — COMMENTED** [review]
I think overall formatting and standards need to be fixed for this repo

**Pull request merged**